src/Shared/NativeMethodsShared.cs

```diff
@@ -818,14 +822,36 @@ internal static DateTime GetLastWriteFileUtcTime(string fullPath)
             DateTime fileModifiedTime = DateTime.MinValue;
             if (IsWindows)
             {
-                WIN32_FILE_ATTRIBUTE_DATA data = new WIN32_FILE_ATTRIBUTE_DATA();
-                bool success = false;
+                if (UseSymlinkTimeInsteadOfTargetTime)
+                {
+                    WIN32_FILE_ATTRIBUTE_DATA data = new WIN32_FILE_ATTRIBUTE_DATA();
+                    bool success = false;
```

Comment:
This was a fun conflict with myself. I just wrapped the symlink code in `if windows`. Filed #826 to make sure it works right on *nix.